Pcap++/header/DpdkDeviceList.h

```diff
@@ -194,7 +194,7 @@ namespace pcpp
 		 * worker threads couldn't be run
 		 */
 		bool startDpdkWorkerThreads(CoreMask coreMask, std::vector<DpdkWorkerThread*>& workerThreadsVec);
-
+		bool startDpdkWorkerThread(CoreMask coreMask, DpdkWorkerThread* worker);
```

Comment:
`coreMask` seems redundant since `rte_eal_mp_remote_launch` runs on all cores, am I missing something?

Also - I'd add a boolean parameter: `bool runOnMasterCore` to determine whether to use `CALL_MASTER` or `SKIP_MASTER`